vars/skipStage.groovy

```diff
@@ -78,7 +78,9 @@ boolean skip_ftest(String distro, String target_branch) {
 }
 
 boolean skip_ftest_valgrind(String distro, String target_branch) {
-    if (! skip_stage_pragma('func-test-vm-valgrind', 'true')) {
```

Comment:
It's strange.  I would have thought that this already skipped the stage by default by passing `'true'` as the default value:
https://github.com/daos-stack/pipeline-lib/blob/293825867929a6bcb544d82514ef49b1b35ef39e/vars/skipStage.groovy#L14-L16

`cachedCommitPragma()` should return it's second argument if the pragma is not specified.